test/contrib/airtable/test_airtable_datasource.py

```diff
@@ -0,0 +1,40 @@
+import os
+from dataclasses import dataclass
+from test.tags import integration_test
+
+from django.conf import settings
+from django.test import TestCase
+
+from groundwork.contrib.airtable import datasources
+
+
+@integration_test
+class AirtableApiTests(TestCase):
+    def setUp(self):
+        self.datasource = datasources.AirtableDatasource(
+            resource_type=MyResource,
+            api_key=settings.EXAMPLE_AIRTABLE_API_KEY,
+            base_id=settings.EXAMPLE_AIRTABLE_BASE,
+            table_name="Table 1",
```

Comment:
`table_name` could also go on the env as `settings.EXAMPLE_AIRTABLE_TABLE_NAME`? Appreciate that it is the default table name, but it can be changed by the user of course.

Reply:
Main reason is that it introduces another environmental variable that needs to be shifted between different environments that this needs to run in (local dev machines, ci etc). Secondary reason – we might plausably introduce an integration test for multiple tables (if we wanted to test relationships, for example) so making it explicit in the code which table is being referenced in a test is useful.